src/models/JobCandidate.js

```diff
@@ -49,7 +49,12 @@ module.exports = (sequelize) => {
       userId: {
         field: 'user_id',
         type: Sequelize.UUID,
-        allowNull: false
+        allowNull: true // temporarly mark it as nullable until complete removal
```

Comment:
Please add a subtask to the Jira story for this work to remove `userId` and change the nullable for `tc_user_id`

Reply:
Done https://topcoder.atlassian.net/browse/TSJR-325
It is added into Sprint 8